src/remote/utils.go

```diff
@@ -464,6 +464,18 @@ func convertPlatform(config *core.Configuration) *pb.Platform {
 	return platform
 }
 
+// targetPlatform returns the platform properties for a target, including any global ones.
+func (c *Client) targetPlatform(target *core.BuildTarget) *pb.Platform {
```

Comment:
nit: This threw me for a sec. Thought it had something to do with cross compiling. Maybe `targetPlatformProperties()`.